patches/net-proxy_resolution-configured_proxy_resolution_service.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/net/proxy_resolution/configured_proxy_resolution_service.cc b/net/proxy_resolution/configured_proxy_resolution_service.cc
-index 7468d23f9e2e92e27f9eb05b934ca2c8a5470010..aa038cd2a973e642bb1651ab26d95d49f84518d7 100644
+index 7468d23f9e2e92e27f9eb05b934ca2c8a5470010..ec72a569a42c84381b5058a33cbb39bfa1390535 100644
 --- a/net/proxy_resolution/configured_proxy_resolution_service.cc
 +++ b/net/proxy_resolution/configured_proxy_resolution_service.cc
-@@ -977,6 +977,7 @@ int ConfiguredProxyResolutionService::TryToCompleteSynchronously(
-   config_->value().proxy_rules().Apply(url, result);
-   result->set_traffic_annotation(
-       MutableNetworkTrafficAnnotationTag(config_->traffic_annotation()));
-+  BRAVE_TRY_TO_COMPLETE_SYNCHRONOUSLY
- 
-   return OK;
- }
+@@ -921,6 +921,7 @@ int ConfiguredProxyResolutionService::ResolveProxy(
+   // Check if the request can be completed right away. (This is the case when
+   // using a direct connection for example).
+   int rv = TryToCompleteSynchronously(url, result);
++  BRAVE_RESOLVE_PROXY
```

Comment:
`BRAVE_CONFIGURED_PROXY_RESOLUTION_SERVICE_RESOLVE_PROXY`

Reply:
fixed and squashed.